x-pack/plugin/eql/src/main/java/org/elasticsearch/xpack/eql/plugin/TransportEqlSearchAction.java

```diff
@@ -41,7 +40,7 @@
 
     @Inject
     public TransportEqlSearchAction(Settings settings, ClusterService clusterService, TransportService transportService,
-            ThreadPool threadPool, ActionFilters actionFilters, PlanExecutor planExecutor) {
+                                    ThreadPool threadPool, ActionFilters actionFilters, PlanExecutor planExecutor) {
```

Comment:
Same as above, formatting on non-modified lines is noise.

Reply:
this is the formatting misalignment that I missed in my previous PR, now trying to fix it